### ISSUE TITLE
Implement a history method

### ISSUE THREAD

**Issue by pavanramkumar** [glm-tools/pyglmnet#134]
It should return a history object (maybe `dict`) that has the loss function and other summary stats about the parameters for each iteration. 

We can look into keras or GRLM for style:
http://keras.io/callbacks/
https://github.com/cehorn/GLRM/blob/master/glrm/convergence.py

**Comment by jasmainak** [comment]
how do you envision this to be different from the logger that we already have?

**Comment by pavanramkumar** [comment]
does logger give us loss and log likelihood for each iteration that we can later visualize?

**Comment by jasmainak** [comment]
okay, but why not simply have `glm.loss` and `glm.log_likelihood` ?

**Comment by cxrodgers** [comment]
I wrote something that achieves a similar purpose in my own fork. I just created a PR for comments/review. It may not be exactly what you had in mind.
https://github.com/glm-tools/pyglmnet/pull/287

**Comment by pavanramkumar** [comment]
closing see #316
